components/scream/src/share/atm_process/atmosphere_process_group.cpp

```diff
@@ -190,6 +190,106 @@ gather_internal_fields  () {
   }
 }
 
+bool AtmosphereProcessGroup::
+are_column_conservation_checks_enabled ()
+{
+  // Loop through processes and return true if an instance is found.
+  for (auto atm_proc : m_atm_processes) {
+
+    // Recursively check processes of internal groups. If one is found, return true,
+    // else continue to the next process.
+    auto atm_proc_group = std::dynamic_pointer_cast<AtmosphereProcessGroup>(atm_proc);
+    if (atm_proc_group) {
+      if (atm_proc_group->are_column_conservation_checks_enabled()) {
+        return true;
+      } else {
+        continue;
+      }
+    }
+
+    // If this process is not a group, query enable_column_conservation_checks
+    // and return true if true.
+    if (atm_proc->get_params_non_const().get<bool>("enable_column_conservation_checks", false)) {
```

Comment:
You could do
```
if (not atm_proc->get_params().isParameter("enable_column_conservation_checks") ||
    not atm_proc->get_params().get<bool>("enable_column_conservation_checks")) {
    return true;
 }
```
C++'s boolean short-circuit will ensure the get method is not called if the param is not stored.